components/content/ArticlesListItem.vue

```diff
@@ -24,9 +24,11 @@ defineProps({
 
 <template>
   <article v-if="article._path && article.title" :class="{ 'featured': featured }">
-    <NuxtLink :to="article._path">
-      <img v-if="article.cover" :src="article.cover">
-    </NuxtLink>
+    <div class="image">
+      <NuxtLink :to="article._path">
+        <img v-if="article.cover" :src="article.cover">
```

Comment:
We may want to also add the `alt` here

```suggestion
        <img v-if="article.cover" :src="article.cover" :alt="article.title">
```